herald/src/app/+home/home.component.ts

```diff
@@ -1,15 +1,51 @@
 import {Component, OnInit} from '@angular/core';
+import {Http} from '@angular/http';
+
+import {Subject} from 'rxjs/Rx';
+import {PlayerDataStore} from '../shared/player-data/player-data-store.component';
 
 @Component({
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.css']
 })
 export class HomeComponent implements OnInit {
 
-  constructor() {
+  playerSearchResults: PlayerDataStore;
+  maxSearchResults: number = 50;
+  searchDataSubject: Subject<any> = new Subject<any>(); //to provide data to the player search table
+
+  constructor(private http: Http) {
   }
 
   ngOnInit() {
+    this.playerSearchResults = new PlayerDataStore(this.http);
+  }
+
+  /**
+  * does a search for players with the value provided
+  * @param value the player which is being searched for
+  */
+  doPlayerSearch(value: string){
```

Comment:
We shouldn't have API connections all throughout our components.  Shouldn't this be moved to a service class?